xtp/src/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
 add_subdirectory(libxtp)
 add_subdirectory(tools)
-add_subdirectory(tests)
+if(ENABLE_TESTING)
+  add_subdirectory(tests)
```

Comment:
I think we are testing for enable_testing in the directory now.

Reply:
Changed that and indeed the tests were not build since ENABLE_TESTING was not defined. The test_orca.cc had some issues with the QMPacakgeFactory call due to the initial merge. I fix that as well